### PR TITLE
Lock snapcraft to v5 since v7 breaks the build

### PR DESCRIPTION
**Pull request by vraravam**
versions available at: https://snapcraft.io/snapcraft

#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Description of Change

Trying to fix the build by locking on v5.x (prior to v7, the latest version available is v5: https://snapcraft.io/snapcraft)

#### Motivation and Context

Get successful nightly builds.

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)
- [x] `npm test` passes
- [ ] I tested/previewed my changes locally

### CODE REVIEW

File: .github/workflows/builds.yml [hunk 1]
```diff
@@ -261,7 +261,7 @@ jobs:
           CSC_IDENTITY_AUTO_DISCOVERY: false
           SNAPCRAFT_LOGIN: ${{ secrets.SNAPCRAFT_LOGIN }}
         run: |
-          sudo snap install snapcraft --classic
+          sudo snap install snapcraft --channel=5.x/stable --classic
```
Comment (SpecialAro) [inline .github/workflows/builds.yml:264]: it should be v6.1 as per this successful build https://github.com/ferdium/ferdium-app/runs/6872922102?check_suite_focus=true#step:16:20

File: .github/workflows/builds.yml [hunk 2]
```diff
@@ -273,7 +273,7 @@ jobs:
...
```
Comment (SpecialAro) [inline .github/workflows/builds.yml:276]: it should be v6.1 as per this successful build https://github.com/ferdium/ferdium-app/runs/6872922102?check_suite_focus=true#step:16:20

File: .github/workflows/builds.yml [hunk 3]
```diff
@@ -285,7 +285,7 @@ jobs:
...
```
Comment (SpecialAro) [inline .github/workflows/builds.yml:288]: it should be v6.1 as per this successful build https://github.com/ferdium/ferdium-app/runs/6872922102?check_suite_focus=true#step:16:20

**Review by SpecialAro — APPROVED** [review]
LGTM, should fix. Though, we should try to fix it for v7.x when we have the time

File: .github/workflows/builds.yml (hunk 1 above)
Comment (NathanaelGandhi) [reply .github/workflows/builds.yml:264]: V6 isnt listed at [https://snapcraft.io/snapcraft](https://snapcraft.io/snapcraft)
image

Comment (SpecialAro) [reply .github/workflows/builds.yml:264]: > V6 isnt listed at https://snapcraft.io/snapcraft image

Weird :o

@NathanaelGandhi , by any chance do you have time to adapt our build to 7.x today?

Comment (vraravam) [reply .github/workflows/builds.yml:264]: Yes - this page doesn't list any `6.x` - thus, I left it at `5.x`

**Pull request merged**